Task1/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Nine task</title>
+        <link href="style.css" rel="stylesheet" type="text/css">
+    </head>
+    <body>
+        <h1>Tic Tac Toe</h1>
+
+        <div class="title" id="title">
+            Move x player
+        </div>
+
+        <table class="table">
+            <tr class="row">
+                <td class="colomn" onclick="сlick(this, 0, 0)"></td>
```

Comment:
I think better use an event handler instead of this attribute. and for set position for this element you can use attribute data-x and data-y. Because you need to separate js and html